lensless/__init__.py

```diff
@@ -22,6 +22,10 @@
     GradientDescentUpdate,
 )
 
+from .trainable_recon import TrainableReconstructionAlgorithm
```

Comment:
have you tried using `LenslessPiCam` in an environment which doesn't have PyTorch? I'm wondering if this would fail since `trainable_recon` throws an error if it cannot import torch. In that case, I would put this under a try-except as with APGD/Pycsou.